sdk/python/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items.ipynb

```diff
@@ -639,6 +654,87 @@
    ]
   },
   {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4.2.1 Individual runs with models from Hugging Face (Preview)\n",
+    "\n",
+    "In addition to the model algorithms supported natively by AutoML, you can launch individual runs to explore any model algorithm from HuggingFace that supports image classification. Please refer to this [documentation](https://huggingface.co/models?pipeline_tag=image-classification) for the list of models.\n",
+    "\n",
+    "While you can use any model from Hugging face to support this task, we have curated a set of model algorithms in our registry. We provide a set of sensible default hyperparameters for these model algorithms. You can fetch the list of curated model algorithms using code snippet below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ml_client = MLClient(credential, registry_name=\"azureml-staging\")\n",
```

Comment:
this will overwrite the ml_client variable and may cause subsequent runs to fail because the ml_client is not connected to a workspace. maybe change the name to registry_ml_client or similar

Reply:
applies to both all notebooks

Reply:
Good catch. Will update it.